src/languages/javascript.js

```diff
@@ -263,7 +263,7 @@ export default function(hljs) {
         beginKeywords: 'constructor', end: /\{/, excludeEnd: true
       },
       {
-        begin:'(get|set)\\s*(?=' + IDENT_RE+ '\\()',
+        begin: "(get|set)\\s+(?=" + IDENT_RE + "\\()",
```

Comment:
Please don't change the string type for no reason.

Reply:
N/m, I'll take this back.  This kind of feedback would drive me nuts since we don't have clear guidelines.   As long as you aren't changing the whole file I'm good.